### PR TITLE
@accounts/mongo : Monorepo integration

### PR DESCRIPTION
**Pull request by Aetherall**

### CODE REVIEW

File: packages/database-mongo/tslint.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (pradel) [inline packages/database-mongo/tslint.json]: you can remove this file

File: packages/database-mongo/package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  "typings": "lib/index.d.ts",
+  "scripts": {
+    "start": "tsc --watch",
+    "compile": "tsc",
+    "testonly": "jest",
+    "test:watch": "jest --watch",
+    "coverage": "yarn testonly --coverage",
+    "prepublishOnly": "yarn compile"
+  },
+  "renovate": {
```
Comment (pradel) [inline packages/database-mongo/package.json]: can you remove renovate config also please?

File: packages/database-mongo/tslint.json (hunk 1 above)
Comment (Aetherall) [reply packages/database-mongo/tslint.json]: oops done :)

File: packages/database-mongo/package.json (hunk 2 above)
Comment (Aetherall) [reply packages/database-mongo/package.json]: also done, along with the circleci folder and the .gitignore

**Comment by pradel** [comment]
Good to merge @Aetherall

**Pull request merged**